### PR TITLE
Update locksmithWorker image

### PR DESCRIPTION
**Pull request by aevox**
Updated locksmithWorker image. vault-fernet-locksmith now needs a
configuration file. It is added in docker-images config.

### CODE REVIEW

File: pkgs/docker-images/config/locksmith/config.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+      {{ with secret "secret/locksmith" -}}
+      consul-token: {{ .Data.consulToken }}
+      {{- end }}
+    '';
+  };
+
+  locksmithPreStart = ''
+    consul-template-wrapper -- -once \
+      -template "${locksmithConf}:/run/consul-template-wrapper/etc/locksmith/config.yaml"
```
Comment (eonpatapon) [inline pkgs/docker-images/config/locksmith/config.nix]: How do you make sure /run/consul-template-wrapper/etc/locksmith/ exists ?

**Comment by nlewo** [comment]
Could you change the commit message according to `nixpkgs` convention:
```
locksmith: 1.1 -> 2.0

Detailled message here

```

File: pkgs/docker-images/config/locksmith/config.nix (hunk 1 above)
Comment (nlewo) [reply pkgs/docker-images/config/locksmith/config.nix]: Can you check consul-template-wrapper creates the dir if it doesn't exist (I think it is the case)?

**Comment by aevox** [comment]
Sure ! It is done.

**Pull request merged**